docs/content/getting-started/built_in_templates.md

```diff
@@ -0,0 +1,108 @@
+# Using Templates
+
+Blueprint comes with several templates to help you get started with your project.
+
+> Note: We are continuing to add additional templates. Please check back periodically.
+
+To install from a template, specify the template name after the initial flake init 
+command, preceded by a hash symbol. For example, to use the template called system 
+manager, type:
+
+```
+nix flake init -t github:numtide/blueprint#system-manager
+```
+
+## Default Template
+
+Init command:
+
+```bash
+nix flake init -t github:numtide/blueprint
+```
+
+This is a bare-bones project as described in [getting started](../getting-started/install.md).
+
+## NixOS and Darwin Shared Homes Template
+
+```
+nix flake init -t github:numtide/blueprint#nixos-and-darwin-shared-homes
+```
+
+This template is a bit of an example plus a template. You'll want to study all the
+files carefully. It shows how you can define and reuse modules, in this case nixos
+and home-manager.
+
+Look carefully at the folder structure; in this case we're using `hosts` and
+`modules` folders which are both picked up by Blueprint.
+
+If you drill down into the folders, you'll see inside the `hosts` folder, are a
+`my-darwin` folder and a `my-nixos` folder, both of which are imported by Blueprint.
+This defines the two hosts called `my-darwin` and `my-nixos`.
+
+Their respective configuration files both import a shared 
+`modules/nixos/host-shared.nix` module between them.
+
+Also, both hosts define a `me` user and their home-managed configuration
+simply imports `modules/homes/home-shared.nix`.
+
+Finally, notice in the root flake.nix we're adding the home-manager and nix-darwin 
+inputs, which serve as dependencies for managing home configurations and macOS 
+integrations, respectively.
+
+The idea with this template is that you can use this example to get started on
+how to share configurations between different system and home environments on different hosts.
+
+
+## Toml-DevEnvs
```

Comment:
Might not hurt to include it, but it's a comparatively niche use-case IMO, and I'd rather just link to devshells README as you do with system-manager below.